client/my-sites/ads/main.jsx

```diff
@@ -223,18 +222,30 @@ const AdsMain = React.createClass( {
 } );
 
 export default connect(
-	( state, ownProps ) => ( {
-		requestingWordAdsApproval: isRequestingWordAdsApprovalForSite( state, ownProps.site ),
-		wordAdsError: getWordAdsErrorForSite( state, ownProps.site ),
-		wordAdsSuccess: getWordAdsSuccessForSite( state, ownProps.site ),
-		isUnsafe: isSiteWordadsUnsafe( state, ownProps.site.ID ),
-		isRequestingWordadsStatus: isRequestingWordadsStatus( state, ownProps.site.ID )
-	} ),
+	( state ) => {
+		const site = getSelectedSite( state );
+		const siteId = getSelectedSiteId( state );
+
+		return {
+			site,
+			siteId,
+			siteSlug: getSelectedSiteSlug( state ),
+			requestingWordAdsApproval: isRequestingWordAdsApprovalForSite( state, site ),
+			wordAdsError: getWordAdsErrorForSite( state, site ),
+			wordAdsSuccess: getWordAdsSuccessForSite( state, site ),
+			isUnsafe: isSiteWordadsUnsafe( state, siteId ),
+			isRequestingWordadsStatus: isRequestingWordadsStatus( state, siteId )
+		};
+	},
```

Comment:
This is optional, but this is getting long enough that we should create `mapStateToProps`, `mapDispatchToProps` and `mergeProps` variables that we pass to connect.